lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -191,3 +193,22 @@ def psfont(*args, **kwargs):
     ax.text(0.5, 0.5, 'hello')
     with tempfile.TemporaryFile() as tmpfile, pytest.raises(ValueError):
         fig.savefig(tmpfile, format='pdf')
+
+
+def test_pdf_savefig_when_color_is_none():
+    backup_params = mpl.rcParams.copy()
+    mpl.rcParams.update(mpl.rcParamsDefault)
+    plt.subplot()
+    plt.axis('off')
+    plt.plot(np.sin(np.linspace(-5, 5, 100)), 'v', c='none')
+    try:
```

Comment:
I think this try/except block can go; if the next line throws and exception the test will automatically fail.